domains/rishav.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "portfolio domain",
+    "repo": "https://github.com/r15hav/r15hav.github.io",
+    "owner": {
+        "username": "r15hav",
+        "email": "r15hav@gmail.com",
```

Comment:
just noticed this, remove the comma here

Reply:
```suggestion
        "email": "r15hav@gmail.com"
```